modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
 +
-[source,posh]
-----
-# Powershell script to configure SSH on vSphere Windows VMs
-Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
-$firewallRuleName = "ContainerLogsPort"
-$containerLogsPort = "10250"
-New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
-Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-Install-Module -Force OpenSSHUtils
-Set-Service -Name ssh-agent -StartupType 'Automatic'
-Set-Service -Name sshd -StartupType 'Automatic'
-Start-Service ssh-agent
-Start-Service sshd
-$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
-$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
-$passwordConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$authFileConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys','#AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys'
-$authFileConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$pubKeyLocationConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'Match Group administrators','#Match Group administrators'
-$pubKeyLocationConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-Restart-Service sshd
-New-item -Path $env:USERPROFILE -Name .ssh -ItemType Directory -force
-----
+[IMPORTANT]
+====
+The Windows Server 2004 VM must include the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 1809` VM image.
+====
 
 . Install and configure VMware Tools version 11.0.6 or greater on the Windows VM. See the link:https://docs.vmware.com/en/VMware-Tools/index.html[VMware Tools documentation] for more information.
 
 . After installing VMware Tools on the Windows VM, verify the following:
+
 .. The `C:\ProgramData\VMware\VMware Tools\tools.conf` file has the following entry:
 +
 [source,ini]
 ----
 exclude-nics=
 ----
 +
-This entry ensures the following:
+This entry ensures the cloned vNIC generated on the Windows VM by the hybrid-overlay is not ignored.
+
+.. The Windows VM has a valid IP address in vCenter:
++
+[source,posh]
+----
+C:\> ipconfig
+----
+
+.. The VMTools Windows service is running:
++
+[source,posh]
+----
+C:\> Get-Service -Name VMTools | Select Status, StartType
```

Comment:
This is a PowerShell command, and `ipconfig` isn't. There should be some distinction.

"posh" in `[source,posh]` means PowerShell?

Reply:
@jrvaldes what is `ipconfig` being executed from? Would it be more accurate as this?
```
[source,terminal]
$ ipconfig
```

I assumed since we were checking the ipconfig in the Windows VM, that it was from PowerShell. (And yes, `posh`=PowerShell in Asciidoc).

Reply:
Correct, `ipconfig` is a Windows Command Prompt  (cmd.exe), however it works in PowerShell. Not the other way around, `Get-Service -Name VMTools | Select Status, StartType`  does not work in cmd.exe.

Prefixing PowerShell commands with `PS C:\>` may help, for example:

`PS C:\> Get-Service -Name VMTools | Select Status, StartType`